SimplenoteUITests/SimplenoteUITests.swift

```diff
@@ -0,0 +1,410 @@
+//
+//  SimplenoteUITests.swift
+//  SimplenoteUITests
+//
+//  Created by Xue Qin on 1/31/18.
+//  Copyright © 2018 Automattic. All rights reserved.
+//
+
+import XCTest
+
+class SimplenoteUITests: XCTestCase {
+    
+    let app = XCUIApplication()
+    let username = "ui.privacy.utsa4@gmail.com"
+    
+    override func setUp() {
+        super.setUp()
+        
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        app.launch()
+        
+        if app.staticTexts["Welcome to Simplenote"].exists {
```

Comment:
I think we need to completely reset the app here, as the welcome screen doesn't show again if you sign out. It only shows one time ever to the user.